IRCT-CL/src/main/java/edu/harvard/hms/dbmi/bd2k/irct/cl/util/Utilities.java

```diff
@@ -224,7 +232,13 @@ public static String extractUserFromTokenIntrospection(HttpServletRequest req, S
 
 		ObjectMapper json = IRCTApplication.objectMapper;
 		CloseableHttpClient client = IRCTApplication.CLOSEABLE_HTTP_CLIENT;
-		HttpPost post = new HttpPost(token_introspection_url);
+
+		String[] token_intro_urlArray = token_introspection_url.substring(8).split("/");
```

Comment:
You can just do: String token_intro_url = token_introspection_url.substring(8).split("/")[0];  
As in below code you are just using 0 index only. So, if other index data is not required or will not be required in future, than no need to maintain an Array.